examples/pipelineruns/pipelinerun.yaml

```diff
@@ -16,7 +16,7 @@ spec:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: default-cluster-admin
```

Comment:
> Totally +1 on the spirit of the change, but I think it uncovers some things we'll want to clean up. Some of these PipelineResources should just be embedded in the Pipeline that uses them, which would make a lot of this cleaner.

Good point. I'm trying to provide a quick solution to this because right now we are not running the taskrun examples anymore, which is pretty bad.
Embedding PipelineResources would be nice, but I'd rather do it on the next iteration.

Another approach to solve the problem could be to create a namespace for every example, which - with the exception of clustertasks - would render all resource name unique.